### DISCUSSION TITLE
jumper t-lite and top rc Spitfire

### DISCUSSION THREAD

**Discussion by Pappaburn123** [EdgeTX/edgetx#4536] [Q&A]
can i fly top rc Spitfire with my jumper t-lite radio if so how ?

**Comment by pfeerick** [comment]
Exactly what plane is that... is the same one? => https://www.ebay.com.au/itm/175536254539

If that is the right plane, it looks like it may be a rebadged / go under the Eachine name also... and those are known to be work with the V761 protocol ([more details on that here](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/Protocols_Details.md#v761---48)). There is even specific mention of Top RC Hobby Spitfire there also as a subprotocol choice for V761 😉 

Is your Jumper T-Lite a MPM version (and if so, is the the full 4IN1, and NOT the Lite/CC2500 version?)? If I'm looking at the right module, it needs the NRF24L01 RF chip, so won't work with the CC2500-only MPM. 

If you've been able to say yes to all of that, there is a good chance you'll be able to get it to work :)

**Reply by Pappaburn123** [reply]
how can i see if i got the cc2500 or the full 4in1 version ? igot the V761 protco l 3channel 4channel but not top rc subprotocol choice

**Reply by pfeerick** [reply]
If you don't know from when you got it, the only way to tell AFAIK is to see if the MPM lets you select that protocol. For that to work, first you just need to double check the "Status" line does **not** say "No Telemetry" and instead shows the version number and channel mapping order for the MPM. If it was to say "No Telemetry", this means the MPM isn't talking to the radio. 

Since you're are getting the V761, you probably have a 4IN1. If you had the CC2500 and the MPM is working properly, it will limit the list of protocols to those it can actually use. If you're only getting the 3CH/4CH subprotocols, but not the TOPRC one, you probably need to update your firmware... I think it was only recently added or changed, and is definitely in V1.3.3.33 (the current MPM firmware version). You can download it from https://downloads.multi-module.org/?jp-tlt4in1

Save the file to your radio somewhere - I usually create a MPM folder in the FIRMWARE folder on my radios. Then go into SYS -> SD CARD, browse for that file, and when you select it, choose to update the internal MPM. Once that is done, you should have the TOPRC subprotocol as an option. If it still won't show, make sure you're on the latest stable version of EdgeTX (v2.9.2) as there was one or two versions of 2.8.x  that wouldn't show *all* of the MPM protocols/subprotocols.

**Reply by Pappaburn123** [reply]
i did as you said but i cant update only says copy move or delete but thank you for trying to help me. 
im useing edge v2,9,2 after subtupe 3channel and 4channel there is blank and status say prot invalid

**Reply by pfeerick** [reply]
Yeah, it will say that until you can update the MPM firmware... mine was doing the same and I think the firmware on it was from within the last six months until I updated it. This was with the `mm-stm-serial-aetr-v1.3.3.33.bin` (assuming you went for the default AETR channel order firmware)? As you should have "Flash Int. Multi, Flash Ext. Multi, Copy, Rename, Delete" as your options when you long press that file in the SD browser on the radio?

**Reply by Pappaburn123** [reply]
BIG THANK YOU!!!! it works now im so happy :)

**Reply by pfeerick** [reply]
Great to hear! :) Happy flying!
